### PR TITLE
added install script, function is as follows

### PR DESCRIPTION
**Pull request by rhysmorgan134**
*contains a list of dependencies
*installs dependencies
*makes ilclient for raspberry pi
*creates directories
*makes the application
*creates UDEV permissions (creates an open rule, user may want to create a more secure rule)
*starts application

### CODE REVIEW

**Comment by rhysmorgan134** [comment]
I have successfully used the install script various times now, added some extra features including creating udev rules if they dont exist.

Any new dependices just need adding to the list at the top of the script

**Comment by rhysmorgan134** [comment]
Also added walkthrough video from youtube, and small text version to readme

**Comment by rsjudka** [comment]
thanks for doing this! i've used it before and it definitely makes the process a lot easier, video walkthrough is also a nice touch

It does seem very pi-specific, but for now as that's what most people want to be using i think it'll be fine

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```
Comment (rsjudka) [inline install.sh:1]: you should be able to actually commit this file as an executable, would help to remove that step from the readme

Comment (rhysmorgan134) [reply install.sh:1]: You are right, the install.sh was commited as an executable, just put it there to cover. Removed the command from README section.

Also added "Raspberry Pi only" to the header. I will look into modifying for non Raspberry Pi systems in the future.

**Comment by rsjudka** [comment]
lgtm, I want to run the script again on a fresh image to make sure it's consistent, and if that goes well I'll approve and merge 👍

**Pull request merged**